packages/editor/src/components/preferences-modal/index.js

```diff
@@ -59,212 +59,254 @@ export default function EditorPreferencesModal( { extraSections = {} } ) {
-									'Adds a category with the most frequently used blocks in the inserter.'
+									'Highlights the current block and fades other content.'
 								) }
-								label={ __( 'Show most used blocks' ) }
+								label={ __( 'Spotlight mode' ) }
 							/>
+							{ extraSections?.appearance }
 						</PreferencesModalSection>
-						<PreferencesModalSection
-							title={ __( 'Manage block visibility' ) }
-							description={ __(
-								"Disable blocks that you don't want to appear in the inserter. They can always be toggled back on later."
-							) }
-						>
-							<BlockManager />
-						</PreferencesModalSection>
-					</>
-				),
-			},
-		],
+					),
+				},
+				{
+					name: 'accessibility',
+					tabLabel: __( 'Accessibility' ),
+					content: (
+						<>
+							<PreferencesModalSection
+								title={ __( 'Navigation' ) }
+								description={ __(
+									'Optimize the editing experience for enhanced control.'
+								) }
+							>
+								<PreferenceToggleControl
+									scope="core"
+									featureName="keepCaretInsideBlock"
+									help={ __(
+										'Keeps the text cursor within the block boundaries, aiding users with screen readers by preventing unintentional cursor movement outside the block.'
+									) }
+									label={ __(
+										'Contain text cursor inside block'
+									) }
+								/>
+							</PreferencesModalSection>
+							<PreferencesModalSection
+								title={ __( 'Interface' ) }
+							>
+								<PreferenceToggleControl
+									scope="core"
+									featureName="showIconLabels"
+									label={ __( 'Show button text labels' ) }
+									help={ __(
+										'Show text instead of icons on buttons across the interface.'
+									) }
+								/>
+							</PreferencesModalSection>
+						</>
+					),
+				},
+				{
+					name: 'blocks',
+					tabLabel: __( 'Blocks' ),
+					content: (
+						<>
+							<PreferencesModalSection title={ __( 'Inserter' ) }>
+								<PreferenceToggleControl
+									scope="core"
+									featureName="mostUsedBlocks"
+									help={ __(
+										'Adds a category with the most frequently used blocks in the inserter.'
+									) }
+									label={ __( 'Show most used blocks' ) }
+								/>
+							</PreferencesModalSection>
+							<PreferencesModalSection
+								title={ __( 'Manage block visibility' ) }
+								description={ __(
+									"Disable blocks that you don't want to appear in the inserter. They can always be toggled back on later."
+								) }
+							>
+								<BlockManager />
+							</PreferencesModalSection>
+						</>
+					),
+				},
+				window.__experimentalMediaProcessing && {
+					name: 'media',
+					tabLabel: __( 'Media' ),
+					content: (
+						<>
+							<PreferencesModalSection
+								title={ __( 'General' ) }
+								description={ __(
+									'Customize options related to the media upload flow.'
+								) }
+							>
+								<PreferenceToggleControl
+									scope="core/media"
```

Comment:
I must admit that I don't understand fully these scopes, but it seems fine to me for the sake of experiment.

Reply:
It's essentially just a namespace for grouping similar settings. Other common ones are `core/edit-post` or `core/customize-widgets`.

Reply:
The scope is used for setting preferences for different packages. `core` setting was introduced recently for preferences that are on top of `editor` package (like `edit-post and edit-site`) and they are shared across these packages. Meaning if you make a change in site editor the same preference will be effective for post editor.

Personally I'd add the in `core` scope.. --cc @youknowriad

Reply:
I would have used the `core` scope as these are global preferences too but I don't feel very strongly. I think some normalization about these would be good.

Reply:
Thank you for providing more details about the scope.